### DISCUSSION TITLE
How to search and download workshop items?

### DISCUSSION THREAD

**Discussion by eocron** [SteamRE/SteamKit#1017] [General]
Use case:
1) Search for specific mod in workshop
2) Download it to some folder (just uri will be enough)

Purpose:
Trying to create easy to use rimworld mod downloader, packing everything to json, then just applying to existing instance.
Examples currently lacking this use case.

**Comment by yaakov-h** [comment]
It's been a while since I've looked into this, but I believe [DepotDownloader](https://github.com/SteamRE/DepotDownloader) does this.
